index.html

```diff
@@ -106,20 +104,21 @@
       </div>
     </my-app>
 
-    <script src="/bower_components/babel-polyfill/browser-polyfill.js"></script>
+    <script src="/bower_components/babel-polyfill/browser-polyfill.js" async></script>
     <script>
-      // Global Constants
-      window.APP_NAME = 'My App';
-      window.BASE_URL = '/';
+      (() => {
+        if (window.Polymer) {
+          return;
+        }
 
-      // Setup Polymer options
-      window.Polymer = {
-        dom: 'shadow',
-        lazyRegister: 'max'
-      };
+        window.performance && performance.mark && performance.mark('index.html');
```

Comment:
This will fail linters, you should use something like: `if (window.performance && performance.mark) performance.mark('index.html');` instead.